lib/dockerregistry/inventory_test.go

```diff
@@ -2042,13 +2039,123 @@ func TestRunChecks(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		got := sc.RunChecks(edges, test.checks)
+		got := sc.RunChecks(test.checks)
 		err := checkEqual(got, test.expected)
 		checkError(t, err,
 			fmt.Sprintf("checkError: test: %v (Error Tracking)\n", test.name))
 	}
 }
 
+func TestImageSizeCheck(t *testing.T) {
```

Comment:
You should also add cases for negative image size, what happens when you pass in -max-image-size=0, etc.

Always consider as many edge cases as possible...!